types/types.d.ts

```diff
@@ -1,39 +1,57 @@
-/// <reference types="node" />
+/// <reference types='node' />
 
-import { FastifyPluginCallback } from "fastify";
-import type * as Fastify from "fastify";
+import { FastifyPluginCallback } from 'fastify';
+import type * as Fastify from 'fastify';
 
-declare module "fastify" {
+declare module 'fastify' {
   interface FastifyRequest {
     /** Allows to access or modify the session data. */
     session: Fastify.Session;
+
     /** A session store. */
     sessionStore: Readonly<FastifySessionPlugin.SessionStore>;
+
     /** Allows to destroy the session in the store. */
-    destroySession(callback: (err?: Error) => void): void;
-  }
-  interface Session {
-    sessionId: string;
-    encryptedSessionId: string;
-    /** Updates the `expires` property of the session. */
-    touch(): void;
-    /** Regenerates the session by generating a new `sessionId`. */
-    regenerate(): void;
-    /** sets values in the session. */
-    set(key: string, value: unknown): void;
-    /** gets values from the session. */
-    get<T>(key: string): T;
+    destroySession(callback: (err: Error) => void): void;
   }
+
+  interface Session extends Partial<SessionData> {}
```

Comment:
Why is this made `Partial`? It means e.g. `session.touch()` needs a check that `touch` is defined, which it always is